backend/base/templates/signup.html

```diff
@@ -33,7 +33,7 @@
       width: 100%;
     }
     .error {
-      color: red;
+      color: rgb(218, 10, 10);

```

Comment:
Use hex code instead of RGB code. Jus refer this https://htmlcolorcodes.com/ to know few difference between hex code and RGB code.